projects/canopy/src/lib/accordion/accordion-panel-heading/accordion-panel-heading.component.ts

```diff
@@ -21,6 +21,7 @@ let nextUniqueId = 0;
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class LgAccordionPanelHeadingComponent implements AfterViewChecked {
+  @Input() iconName = '';
```

Comment:
Unless there is a reason for using empty string, I suggest either not initializing the value or using `null`.

Reply:
Just wanted to initialise it, since the iconName will always be a string, but it makes sense using your approach too

Reply:
I highly recommend setting the type of `iconName` to `IconName` if you want to ensure proper icons are used.

Reply:
This doesn't apply anymore, as I used another approach to set a decorative icon.